include/slam_toolbox/visualization_utils.hpp

```diff
@@ -46,7 +46,7 @@ inline visualization_msgs::msg::Marker toMarker(
   marker.color.b = 0.0;
   marker.color.a = 1.;
   marker.action = visualization_msgs::msg::Marker::ADD;
-  marker.lifetime = rclcpp::Duration(0.);
+  marker.lifetime = rclcpp::Duration::from_nanoseconds(0);
```

Comment:
from_nanoseconds doesn't exist in Foxy, that's a master build addition, we need to retarget this to a ros2 branch

Reply:
This branch has been added so you can retarget it

Reply:
Thanks. The target branch is now ros2.